### DISCUSSION TITLE
Can i add a custom size?

### DISCUSSION THREAD

**Discussion by Viper912** [VirtualDrivers/Virtual-Display-Driver#84] [Q&A]
can i add a custom size? i am using a 1200x2000 Sasmung A7 Tab. Also Thanks for this amazing tool

**Comment by bennodxb** [comment]
Same question here.

**Comment by james-s-w-clark** [comment]
You add the option to option.txt, e.g. https://github.com/itsmikethetech/Virtual-Display-Driver/blob/master/Virtual%20Display%20Driver%20(HDR)/option.txt
after that, reinstall the driver

I had to do this for 1440p/4k 120Hz :)
